src/snowcli/cli/common/flags.py

```diff
@@ -1 +1,48 @@
+from __future__ import annotations
+
+import typer
+from snowcli.utils import check_for_connection
+
 DEFAULT_CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
+
```

Comment:
Just to be sure. I know that you have checked checkbox in the PR checklist but I want to confirm that lack of changes in README means that you included all previously used option alternatives for each option.

Reply:
This change does not affect application behaviour